libvncclient/vncviewer.c

```diff
@@ -266,6 +266,8 @@ rfbClient* rfbGetClient(int bitsPerSample,int samplesPerPixel,
   client->destHost = NULL;
   client->destPort = 5900;
   
+  client->connectTimeout = 60;
```

Comment:
shouldn't this use  `SOCKET_DEFAULT_TIMEOUT` ?

Reply:
I thought about this as well but I see an architectural issue here. Sure we could simply include the new `sockets.h` header but making the vncviewer layer (which operates on top of the rfbproto layer) directly "depend" on the sockets layer would violate the implicit layer architecture (vncviewer -> rfbproto -> sockets). Of course we can ignore these academic issues and mix the layers and make all of the part of the public API (and consequently move the contents of the private sockets.h header into the public rfbclient.h header). So it ends up to the general question in which direction we want to develop the API and the architecture.